pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -303,6 +303,25 @@ func volumeMounts(t *miniov2.Tenant, pool *miniov2.Pool, operatorTLS bool, certV
 	return mounts
 }
 
+// Build the startup probe fro MinIO container.
```

Comment:
can you fix the comment?

`// Build the startup probe fro MinIO container.` to `// Build the startup probe for MinIO container.`

Reply:
Oups.